synchronization/src/main/java/de/cyface/synchronization/WiFiSurveyor.java

```diff
@@ -226,6 +226,7 @@ public void stopSurveillance() throws SynchronisationException {
                 return;
             }
             connectivityManager.unregisterNetworkCallback(networkCallback);
+            networkCallback = null;
```

Comment:
forgot this before. This prevents a crash when you log out (which unregisters the network callbacks) and kill the app, which also unregister the callback if it's not null.